### DISCUSSION TITLE
Implement --pattern or --patterns-from options

### DISCUSSION THREAD

**Discussion by mahikeulbody** [borgbase/vorta#810] [Ideas]
It would be nice to add `--pattern` or `--patterns-from` options in addition to `--exclude` and `--exclude-if-present` options already implemented.

It would be easier to "migrate" to Vorta when one has a `patterns-from` file already set to use with borg.

Thanks for the nice work !

**Comment by Hofer-Julian** [comment]
According to https://borgbackup.readthedocs.io/en/stable/usage/create.html this is still experimental. Not sure if it makes sense to include it in Vorta yet...

**Comment by palto42** [comment]
I'm using the `--pattern` option since many month via `borgmatic`and had no issues. In my view it's much easier to use since it allows a mix of include/exclude which can avoid the usage of complex regular expressions (e.g. exclude some path except specific sub-folder).
